src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
for example, and when i will change it calendar must show proper view
```suggestion
    <div class="calendar calendar--start-day-mon calendar--month-length-29">
```